contrib/pg_tde/documentation/docs/index/tde-limitations.md

```diff
@@ -1,8 +1,30 @@
 # Limitations of pg_tde
 
-The following are current limitations of `pg_tde`:
+Limitations of `pg_tde` (applies to release {{release}}):
 
-* System tables, which include statistics data and database statistics, are currently **not encrypted**.
-* The WAL encryption feature is currently in beta and is not effective unless explicitly enabled. It is not yet production ready. **Do not enable this feature in production environments**.
+* PostgreSQL’s internal system tables, which include statistics and metadata, are not encrypted.
+* `pg_createsubscriber` is not supported.
+* WAL shipping standby (warm standby) is not supported with WAL encryption. You can instead use [Streaming Replication with `tde_heap`](../replication.md), which is supported with encrypted WAL.
+
+## WAL encryption tool support
+
+The following tools and extensions in Percona Distribution for PostgreSQL have been tested and verified to work with `pg_tde` WAL encryption:
+
+??? note "Click to expand"
+    * Patroni
+    * `pg_basebackup` (with `--wal-method=stream`)
+    * `pg_resetwal`
+    * `pg_rewind`
+    * `pg_upgrade`
+    * `pg_waldump`
+    * `pgBackRest`
```

Comment:
The tool should probably not be in code format since the command line tool is called `pgbackrest`.